exercises/accumulate/accumulate_test.go

```diff
@@ -38,9 +38,7 @@ func TestAccumulate(t *testing.T) {
 	for _, test := range tests {
 		actual := Accumulate(test.given, test.converter)
 		if fmt.Sprintf("%s", actual) != fmt.Sprintf("%s", test.expected) {
```

Comment:
Ah I'm wrong, many empty strings expands to `[    ]` which is different from `[]`. So I guess the risk is someone might say `["echo echo echo echo"]` instead of `["echo", "echo", "echo", "echo"]`